README.md

```diff
@@ -17,18 +17,20 @@ Estimating the geographical range of a species from sparse observations is a cha
 ## :hatched_chick: Installation for local development
 1. Clone the repository `git clone git@github.com:UMassCDS/ds4cg2024-inaturalist.git`
 
-2. Navigate to cloned project's root, run `git submodule init` and then `git submodule update --remote --merge`
+2. Create .env and .docker.env at the project root. Copy the contents that were given to you on **slack** into these files.
 
-3. You are all set for setting up the code.
+3. Navigate to cloned project's root, run `git submodule init` and then `git submodule update --remote --merge`
+
+4. You are all set for setting up the code.
+
+Note: We need .env for local development, and .docker.env for docker containers. The difference between them is database url, which is caused by how docker manages networks.
```

Comment:
consolidate all the notes into one big Note. or just "additional information" and have them be bullet points with bolding for the main point of each thing.

Reply:
@oz03-hub  can you consolidate these notes into one with just bullet point.